bin/compile

```diff
@@ -56,13 +56,15 @@ debug "STACK: $STACK"
 debug "SOURCE_VERSION: $SOURCE_VERSION"
 
 # Set target org alias
-TARGET_ORG_ALIAS="ra$SOURCE_VERSION"
-TARGET_ORG_NAME="\"Pipeline Review App: $SOURCE_VERSION\""
-
 if [ "$CI" == "true" ]; then
   # CI doesn't include SOURCE_VERSION, so set it to a default value
-  TARGET_ORG_ALIAS="ci$HEROKU_TEST_RUN_COMMIT_VERSION"
-  TARGET_ORG_NAME="\"Pipeline CI: $HEROKU_TEST_RUN_BRANCH $HEROKU_TEST_RUN_COMMIT_VERSION\""
+  TARGET_ORG_ALIAS="ci-$HEROKU_TEST_RUN_COMMIT_VERSION"
```

Comment:
@cwallsfdc Did you update `app.json` so that when calling the test it uses the new convention?

Reply:
I changed in AppHub.  Sorry about changing - thought a little separation was nice.

```"test-setup": "./vendor/sfdx/release.sh ci-$HEROKU_TEST_RUN_COMMIT_VERSION && ./bin/test-setup.sh",```

Reply:
That's cool. Now's the time to make the changes!